nautobot/core/api/schema.py

```diff
@@ -84,7 +84,12 @@ def get_operation(self, *args, **kwargs):
             for param in operation["parameters"]:
                 if param["name"] == "id" and "description" not in param:
                     param["description"] = "Unique object identifier, either a UUID primary key or a natural-key slug."
-
+            # TODO #3024 make depth a valid schema query parameters
```

Comment:
https://github.com/nautobot/nautobot/actions/runs/4663799363/jobs/8255471066 I believe this is it.

Reply:
Looks like instead of `"type": "integer"`, it maybe needs to be `"schema": {"type": "integer"}`, based on looking at the existing schema for fields like `offset` and `limit`.

Reply:
<img width="1349" alt="Screen Shot 2023-04-19 at 10 52 02 AM" src="https://user-images.githubusercontent.com/46973263/233114703-85559e7e-59a3-4876-b3f6-03c5aab5fddb.png">
It seems like it is still not happy with it after I added `"schema": {"type": "integer"}`.